examples/plugins/project.json

```diff
@@ -5,14 +5,13 @@
   "projectType": "library",
   "targets": {
     "build": {
-      "executor": "@nx/esbuild:esbuild",
+      "executor": "@nx/js:tsc",
```

Comment:
Why did we switch the build tool form `esbuild` to `tsc`?

Reply:
in general #835 and in examples for consistency and eventually getting rid of esbuild dependency (when we can drop `bundle-require` in future)

Reply:
is there another way to use `tsc` than change all imports?